booking/src/main/java/org/dcsa/conformance/standards/booking/party/BookingCarrier.java

```diff
@@ -105,16 +103,72 @@ private String generateSchemaValidVesselIMONumber() {
   private void supplyScenarioParameters(JsonNode actionPrompt) {
     log.info("Carrier.supplyScenarioParameters(%s)".formatted(actionPrompt.toPrettyString()));
     var scenarioType = ScenarioType.valueOf(actionPrompt.required("scenarioType").asText());
-    List<String> validHsCodeAndCommodityType = generateValidCommodityTypeAndHSCodes(scenarioType);
-    CarrierScenarioParameters carrierScenarioParameters =
-        new CarrierScenarioParameters(
-            "Carrier Service %d".formatted(RANDOM.nextInt(999999)),
-            generateSchemaValidVesselIMONumber(),
-            "service name",
-            validHsCodeAndCommodityType.get(0),
-            validHsCodeAndCommodityType.get(1),
-            generateValidPolUNLocationCode(),
-            generateValidPodUNLocationCode());
+    CarrierScenarioParameters carrierScenarioParameters = switch (scenarioType) {
+    case REGULAR, REGULAR_SHIPPER_OWNED -> new CarrierScenarioParameters("Example Carrier Service",
```

Comment:
I prefer the previous way of providing a service name other this static `Example Carrier Service` string (applies to more lines than I highlighted)